src/encoded/static/components/viz/Workflow/Graph.js

```diff
@@ -100,48 +101,12 @@ export default class Graph extends React.Component {
         },
         'nodeClassName' : function(node){ return ''; },
         'nodeEdgeLedgeWidths' : [3,5]
-    }
-
-    constructor(props){
-        super(props);
-        this.render = this.render.bind(this);
-        this.componentDidMount = this.componentDidMount.bind(this);
-        this.width = this.width.bind(this);
-        this.height = this.height.bind(this);
-        this.state = {
-            'mounted' : false
-        };
-    }
-
-    componentDidMount(){
-        this.setState({ 'mounted' : true });
-    }
-
-    componentDidUpdate(pastProps){
```

Comment:
We removed ability for Graph component to calculate its own width after mounting, in order to simplify.
Width must always be passed in as a prop now.